packages/openspp/CHANGELOG.md

```diff
@@ -3,3 +3,14 @@
 ## 1.0.0
 
 - Create OpenSPP adaptor
+
+## 1.0.1
```

Comment:
First off can you put the most recent changes first in the log?

Secondly, for future reference, next time you want to update the version, please run `pnpm changeset` from the repo root. That'll ask you some questions and generate a changelog for you. There should be more details in the main readme.